packs/feats/animal-swiftness.json

```diff
@@ -24,7 +24,71 @@
             "remaster": false,
             "title": "Pathfinder Lost Omens: Ancestry Guide"
         },
-        "rules": [],
+        "rules": [
+            {
+                "choices": [
+                    {
+                        "label": "PF2E.Actor.Speed.Type.Land",
+                        "value": {
+                            "increase": 10,
+                            "type": "land"
+                        }
+                    },
+                    {
+                        "label": "PF2E.Actor.Speed.Type.Fly",
+                        "value": {
+                            "increase": 5,
+                            "type": "fly"
+                        }
+                    },
+                    {
+                        "label": "PF2E.Actor.Speed.Type.Swim",
+                        "value": {
+                            "increase": 5,
+                            "type": "swim"
+                        }
+                    },
+                    {
+                        "label": "PF2E.Actor.Speed.Type.Climb",
+                        "value": {
+                            "increase": 5,
+                            "type": "climb"
+                        }
+                    }
+                ],
+                "flag": "speedType",
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpecificRule.Prompt.Speed",
+                "selection": {
+                    "increase": 5,
+                    "type": "fly"
+                }
+            },
+            {
+                "key": "RollOption",
```

Comment:
Can probably use `"rollOption"` on the ChoiceSet instead?

Reply:
Oh, maybe not because it's an object.

Reply:
I tried that initially and it unfortunately didn't work.

Reply:
You might not need to make it an object. Perhaps make the selection just a `config` calling `speedTypes`, and then AdjustModifier the land speed Flat Modifier if you pick land.